### ISSUE TITLE
Remember search phrase in Crates and Playlists feature/view

### ISSUE THREAD

**Issue by mixxxbot** [mixxxdj/mixxx#9522]
Reported by: **[uklotzde](https://github.com/uklotzde)**
Date: 2018-11-18T12:52:12Z
Status: Confirmed
Importance: Wishlist
Launchpad Issue: [lp1803865](https://bugs.launchpad.net/bugs/1803865)
Tags: library

---

Mixxx allows to enter a search phrase to filter the tracks in the selected crate or playlist. When selecting another crate or playlist the search phrase is cleared and all tracks in the selected crate or playlist are shown. It is not possible to enter a search phrase only once and click through different crates or playlists reusing the search phrase.

TODO:
- Preserve the search phrase while switching between different crates and plalists, i.e. use one shared search state for the Crates and Playlists feature respectively
- Apply the current search phrase of the corresponding feature (Crates or Playlists) on every newly selected crate or playlist until the search is cleared manually

To be discussed: In order to provide a consistent user experience we might also use a shared search state for Tracks / Missing Tracks / Hidden Tracks. Those are completely separate features, although they are displayed in a hierarchical structure.

This idea emerged while reviewing a bugfix PR:
https://github.com/mixxxdj/mixxx/pull/1903

**Comment by daschuer** [comment]
I think we can close this bug as won't fix because we now have the individual remembering of a search phrase.

**Comment by ronso0** [comment]
Yes we can close it, though _not_ because it can't or shouldn't be done, but because it's a duplicate of #11186 ?
